.clang-tidy

```diff
@@ -1,5 +1,5 @@
 ---
-Checks:          'clang-diagnostic-*,clang-analyzer-*,readability-*,modernize-*,bugprone-*,misc-*,google-runtime-int,fuchsia-restrict-system-includes,-clang-analyzer-valist.Uninitialized,-clang-analyzer-security.insecureAPI.rand,-clang-analyzer-alpha.*,-readability-else-after-return,-misc-unused-parameters'
+Checks:          'clang-diagnostic-*,clang-analyzer-*,readability-*,modernize-*,bugprone-*,misc-*,google-runtime-int,fuchsia-restrict-system-includes,-clang-analyzer-valist.Uninitialized,-clang-analyzer-security.insecureAPI.rand,-clang-analyzer-alpha.*,-readability-else-after-return,-readability-magic-numbers,-misc-unused-parameters'
```

Comment:
Have we not standardized this? Should we? As in like, here's a script:
```sh
curl <url to common .clang-tidy>
clang-tidy
```

Reply:
Good idea! I'm gonna copy paste this script for later.

Joke's on you though, a lot of these images don't have curl :P

Reply:
We could fix that...we have the technology!